src/smart-components/workflow/remove-workflow-modal.js

```diff
@@ -91,21 +104,29 @@ const RemoveWorkflowModal = ({
           {
             (finalId && !workflow && !fetchedWorkflow)
               ? <FormItemLoader/>
-              : <FormattedMessage
-                id="remove-workflow-modal-text-single"
-                defaultMessage={ `{name} will be removed.` }
-                values={ {
-                  name: <b>{
-                    finalId
-                      ? fetchedWorkflow && fetchedWorkflow.name || workflow && workflow.name
-                      : (<React.Fragment>
-                        { ids.length }&nbsp;
-                        <FormattedMessage id="approval-processes" defaultMessage="approval processes"/>
-                      </React.Fragment>)
-
-                  }</b>
-                } }
-              />
+              : <React.Fragment>
+                <FormattedMessage
+                  id="remove-workflow-modal-text-single"
+                  defaultMessage={ `{name} will be removed{dependenciesMessageValue }` }
+                  values={ {
+                    name: <b>{
+                      finalId
+                        ? fetchedWorkflow && fetchedWorkflow.name || workflow && workflow.name
+                        : (<React.Fragment>
+                          { ids.length }&nbsp;
+                          <FormattedMessage id="approval-processes" defaultMessage="approval processes"/>
+                        </React.Fragment>)
+                    }</b>,
+                    dependenciesMessageValue:
+                        isEmpty(dependenciesMessage()) ? '.' :
+                          <React.Fragment>
+                            &nbsp;from the following applications: <br/><br/>
```

Comment:
There is a static text in the translations. This should be extracted and filled as a variable.

Reply:
Agree - this change was made before the i118N was merged